src/utils/url.ts

```diff
@@ -38,33 +40,80 @@ export const adminCreateWorldSpace = (worldSlug?: string) =>
 export const adminWorldSpacesUrl = (worldSlug?: string) =>
   generatePath(ADMIN_V4_SPACES_1_PARAM_URL, { worldSlug });
 
-export const venueInsideFullUrl = (spaceSlug?: string) =>
-  generatePath(VENUE_INSIDE_PARAM_URL, { spaceSlug });
+type generateAttendeeInsideSpaceUrlParams = {
+  worldSlug?: WorldSlug;
+  spaceSlug?: SpaceSlug;
+  absoluteUrl?: boolean;
+};
 
-export const venueInsideUrl = (spaceSlug: string) => {
-  return `${VENUE_INSIDE_BASE_URL}/${spaceSlug}`;
+// @debt These being optional is a problem waiting to happen. We need a better
+// way of making world / space slug mandatory
```

Comment:
Making them mandatory is part of the Router. This comment, if anywhere, should be there. Also, maybe at the other end, every DB query making sure data is as declared, not just using type assertions (with keywoard `as`), but type guards (with keyword `if`)

```suggestion
```